pages/linux/rpm.md

```diff
@@ -10,6 +10,10 @@
 
 `rpm -qa '{{mariadb*}}'`
 
+- Install rpm package regardless of currently installed versions:
+
+`rpm -U {{some.rpm}} --force`
```

Comment:
some.rpm is really a poor name. While we are here, let's change it to package_name.rpm here and the penultimate one.